.claude/plans/codebase-rules-audit.md

```
@@ -0,0 +1,200 @@
+4. **Clarified ensureLoaded**: Only use after upsertUnique or for progressive deepening, NEVER to load existing instances more deeply
+5. **Refined React guidance**: Clearer examples of avoiding React hooks with Jazz dependencies
+
+---
+
+## Violations Found
+
+### CRITICAL: Jazz Pattern Violations
+
+| File | Line | Issue | Recommendation |
+|------|------|-------|----------------|
+| `packages/app/src/components/game/settings/options/GameOptionsList.tsx` | 39 | ✅ Fixed - now stores option name string |
```

markdownlint-cli2 (0.18.1)

Table column count
Expected: 4; Actual: 3; Too few cells, row will be missing data

(MD056, table-column-count)